src/components/admin/services/Textarea.jsx

```diff
@@ -1,15 +1,29 @@
-const Textarea = ({ object, setObject, label, maxLength }) => {
+import { TextareaCN } from "../../ui/textarea";
+
+const TextArea = ({
```

Comment:
this component shouldn't be needed at all anymore, we can call the shadcn one directly